PyInstaller/depend/utils.py

```diff
@@ -80,9 +80,15 @@ def create_py3_base_library(libzip_filename, graph):
                             fc.write(BYTECODE_MAGIC)
                             if is_py37:
                                 # Additional bitfield according to PEP 552
-                                # zero means timestamp based
-                                fc.write(struct.pack('<I', 0))
-                            fc.write(struct.pack('<II', timestamp, size))
+                                # one means hash based but don't check the hash
+                                fc.write(struct.pack('<I', 0b1))
+                                with open(mod.filename, 'rb') as fs:
+                                    source_bytes = fs.read()
+                                import importlib.util
```

Comment:
Please move this to the top (wrapped by try/except). We try to avoid imports in the code.

Reply:
Done